### PR TITLE
feat: improve logging of failed installs

### PR DESCRIPTION
**Pull request by lewis6991**

### CODE REVIEW

**Comment by clason** [comment]
oof, looks like GH _really_ doesn't like the async installation -- seems to choke the runner...

**Comment by lewis6991** [comment]
An assert failed because I guess an installation took longer than 10seconds, just pushed a fix.

**Comment by clason** [comment]
can you fix the `ci-install.sh` script? I think the macos archive needs to be renamed (`macos-arm64`); that's simpler than rebasing.

File: scripts/ci-install.sh [hunk 1]
```diff
@@ -6,20 +6,20 @@ NVIM_TAG=${NVIM_TAG-nightly}
...
+  wget "https://github.com/neovim/neovim/releases/download/${NVIM_TAG}/nvim-linux64.tar.gz"
   tar -zxf nvim-linux64.tar.gz
   sudo ln -s "$PWD"/nvim-linux64/bin/nvim /usr/local/bin
   rm -rf "$PWD"/nvim-linux64/lib/nvim/parser
   mkdir -p ~/.local/share/nvim/site/pack/nvim-treesitter/start
   ln -s "$PWD" ~/.local/share/nvim/site/pack/nvim-treesitter/start
 elif [[ $os == Darwin ]]; then
-  curl -L https://github.com/neovim/neovim/releases/download/${NVIM_TAG}/nvim-macos-arm64.tar.gz | tar -xz
+  curl -L "https://github.com/neovim/neovim/releases/download/${NVIM_TAG}/nvim-macos-$(uname -m).tar.gz" | tar -xz
   sudo ln -s "$PWD"/nvim-macos/bin/nvim /usr/local/bin
```
Comment (clason) [inline scripts/ci-install.sh:17]: also needs to be `nvim-macos-arm64` (sorry, tried to fix it myself, but conflicts...)

same for the line below

**Comment by clason** [comment]
> Compile parsers: 1m50

😍

**Comment by lucario387** [comment]
```
Fail	||	indent HTML: new line: start_tag.html[6]	
            ./tests/indent/common.lua:80: Incorrect indentation
                       Found:                                       Expected:
                        |<meta charset="UTF-8">                      |<meta charset="UTF-8">
                        |                                            |
                        |<html lang="en">                            |<html lang="en">
                        |  <head>                                    |  <head>
                        |    <meta                                   |    <meta 
                        |      charset="UTF-8"                       |      charset="UTF-8"
              2 vs 4 => |  />                                        |    />
                        |  </head>                                   |  </head>
                        |</html>                                     |</html>
            
            stack traceback:
            	./tests/indent/common.lua:80: in function 'compare_indent'
            	./tests/indent/common.lua:143: in function 'indent_new_line'
            	./tests/indent/common.lua:199: in function <./tests/indent/common.lua:197>
```

Don't have to ping me, I will be on the case for this soon :tm:

**Comment by clason** [comment]
I've just been ignoring this test failure since the beginning ;)

**Pull request merged**